packages/qwik/src/core/v2/client/vnode-diff.ts

```diff
@@ -1061,7 +1072,7 @@ export const vnode_diff = (
         host,
         (component$Host || container.rootVNode) as HostElement,
         component as OnRenderFn<unknown>,
-        jsxValue.props
+        getMergedProps(currentJSX.props)
```

Comment:
This breaks referencing props inside inline components:

https://qwikdev-build-v2-fix-inline.qwik-8nx.pages.dev/examples/reactivity/counter/#f=Q0o0JsYW2BKNDrzKxZt8IP4MzgCWhkB%2FVoPL6VorCG0FqQpq4b5HrpnVobVGMVCrug6kwtdESmug5oGdM7jYh5pqow8WA0YHMWkWm12gKALZAW2AgD1nC%2FenBsj7IGcrGCiA0jVGdIGKQHjAgr0MVm9bDTZJD%2BJKRNAXIEUKpPGEaDNUQxyKpFFbu9YOLAlvacGjEGqQjT7MAaA4HIkZz8KSLhkPnOpSi6if6wA

should output "render show 0" but outputs "render show [WrappedSignal INVALID [object Object]]"

Let's add a test case for props inside inline :)